pkg/constants/constants.go

```diff
@@ -243,3 +243,11 @@ const (
 	// multicluster field value for context field.
 	LogContextMulticluster = "multicluster"
 )
+
+// Retry Policy
+const (
+	EnableRetryKey   = "openservicemesh.io/retrypolicy/enabled"
+	RetryOnKey       = "openservicemesh.io/retrypolicy/retry-on"
+	NumRetriesKey    = "openservicemesh.io/retrypolicy/num-retries"
+	PerTryTimeoutKey = "openservicemesh.io/retrypolicy/per-try-timeout"
+)
```

Comment:
These constants seem out of place. Should probably reside in `pkg/catalog` or `pkg/trafficpolicy`.

Also each of these annotations needs a comment on what it signifies.